Update API index (#167)
sourcebots/docs: content/api/_index.md

## Running your code

Your code needs to be put in the `zone-0` or `zone-1` folder in a file called
`forklift.py` or `crane.py` for the forklift and crane robots respectively.

{{% notice tip %}}
If this file is missing or incorrectly named, your robot won't do
anything. No log file will be created.
{{% /notice %}}

## Logs

> A log file is saved in the folder so you can see what your robot did,
> what it didn't do, and any errors it raised. Each log file name includes
> the date and time so you can look back at your previous runs.

Reviewer: ```suggestion
A log file is saved in the `output/` folder so you can see what your robot did,
what it didn't do, and any errors it raised. Each log file is in a folder based on its
date and time, so you can look back at your previous runs.
```

## Running your code

Your code needs to be put in the `zone-0` or `zone-1` folder in a file called
`forklift.py` or `crane.py` for the forklift and crane robots respectively.

{{% notice tip %}}
If this file is missing or incorrectly named, your robot won't do
anything. No log file will be created.
{{% /notice %}}

## Logs

A log file is saved in the folder so you can see what your robot did,
what it didn't do, and any errors it raised. Each log file name includes
the date and time so you can look back at your previous runs.

## Included Libraries

Python already comes with plenty of [built-in
libraries](https://docs.python.org/3.9/py-modindex.html) to use. We
install some extra ones which may be of use:

> - [numpy](https://pypi.python.org/pypi/numpy)

Reviewer: Putting the versions in here explicitly is probably a good idea, easier to read.

Also the link here isn't right, it immediately redirects.

Author: fixed
